content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+export PRIMARY_ORG_ID=<PRIMARY_ORG_PUBLIC_ID>
+export USER_EMAIL=<USER_EMAIL>
+export CONNECTION='{"data":{"id":"'${PRIMARY_ORG_ID}'","type":"hamr_org_connections","attributes":{"TargetOrgUuid":"'${DDR_ORG_ID}'","HamrStatus":1,"ModifiedBy":"'${USER_EMAIL}'", "IsPrimary":true}}}'
+
+curl -v -H "Content-Type: application/json" -H 
+"dd-api-key:${PRIMARY_DD_API_KEY}" -H 
+"dd-application-key:${PRIMARY_DD_APP_KEY}" --data "${CONNECTION}" --request POST ${PRIMARY_DD_API_URL}/api/v2/hamr
+```
+{{% /collapse-content %}} <br>
+
+
+
+#### when you have linked the DDR org to your primary org
+{{% collapse-content title=" 5. Create your Datadog API and App key for syncing" level="h5" %}}
+At the secondary Datadog site, create a set of `API key` **and** `App key`. You will use these keys in _steps 7_ to copy dashboards and monitors between Datadog sites. 
+
+<div class="alert alert-info">
+For your Agents, Datadog can copy API key signatures to the secondary backup account for you to prevent you from maintaining another set of API keys for your Agent. Contact your [Customer Success Manager](mailto:success@datadoghq.com) for any questions regarding this.
+</div>
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 6. Configure Single Sign On for the Datadog App" level="h5" %}}
+Go to your [Organization Settings][1] to configure SAML or Google Login for your users. 
+
+**Single Sign On (SSO) is highly recommended** to enable all your users to be able to seamlessly login to your Disater Recovery organization during an outage. 
+
+You must invite your users to your Disaster Recovery organization and give them appropriate roles and permissions. Alternatively, to streamline this operation you can use [Just-in-Time provisioning with SAML][2].
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 7. Set up Resources syncing and scheduler" level="h5" %}}
+Datadog provides a tool called [Datadog sync-cli][3] to copy your dashboards, monitors and other configurations from your primary organization to your secondary organization. You can determine the frequency and timing of syncing based on your business requirements. Regular syncing is essential to ensure that your secondary organization is up-to-date in the event of a disaster. We recommend performing this operation on a daily basis. For information on setting up and running the backup process, see the [datadog-sync-cli README][5]. 
+
+Sync-cli is primarily intended for unidirectional copying and updating resources from your primary org to your secondary org. Resources copied to the secondary organization can be edited, but any new syncing will override changes that differ from the source in the primary organization.
+
+Each item can be added to the sync scope using the sync-cli configuration available in the documentation. Here's an example of a configuration file for syncing specific dashboards and monitors using name and tag filtering from an `EU` site to a `US5` site.
```

Comment:
See if you can do an edit in this section to remove passive voice and to add in some clarity. That way, you can make the actions your user is empowered to do themselves even clearer. Additionally, I think the part about syncing overriding changes could benefit from some simpler phrasing.